lib/graphs.js

```diff
@@ -373,7 +373,7 @@ function changeGraph(action, isStreaming, args) {
     if (sep === '?') { sep = '&'; }
   }
 
-  const requestOptions = mlutil.newRequestOptions(this.client.getConnectionParams(), endpoint, sep, ((action === 'write') ? 'PUT' : 'POST'));
+  const requestOptions = mlutil.newRequestOptions(this.client.getConnectionParams(), endpoint, ((action === 'write') ? 'PUT' : 'POST'));
```

Comment:
Looks like the `sep` code above this and other places in this file can be removed too?

Reply:
Decided against.